acm/odh-core/acm-observability/grafana-dashboards/edge-inference-health.json

```diff
@@ -188,34 +362,74 @@
           "value": "near-edge-aws-ocp412"
```

Comment:
This (and the line above it) appear to still reference a specific cluster. I wonder if it's possible for them to reference something more generic?

Reply:
This is showing the current selected option, which I am guessing would be the default if it exists. I changed it to `local-cluster` but you can't get rid of it from what I can see